compiler/ballerina-parser/src/main/resources/syntax_diagnostic_message.properties

```diff
@@ -235,7 +235,8 @@ error.more.field.match.patterns.after.rest.field=more field match patterns after
 error.action.as.a.wait.expr=''wait'' cannot be used with actions
 error.invalid.usage.of.var='var' is not a valid type descriptor
 error.more.match.patterns.after.rest.match.pattern=more match patterns after rest match pattern
-error-match-pattern-not-allowed=match pattern not allowed
+error.match.pattern.not.allowed=match pattern not allowed
+error.at.least.one.match.clause.required=error at least one match clause required
```

Comment:
shall we reword the error message?
"match statement is required to have one or more match clauses"

Reply:
updated